src/main/resources/db/migration/V0__base_version.sql

```diff
@@ -84,7 +84,7 @@ create table inventory (
    inventory_id integer NOT NULL AUTO_INCREMENT,
    ingredient_id integer ,
    purchase_date timestamp,
-   expiration_date timestamp,
+   expiration_date timestamp  NOT NULL,
```

Comment:
Remove this change.

Reply:
It doesn't work. I am using cloud DB temporarily.